### DISCUSSION TITLE
1px line showing banner in profile screen when you click to hide cover

### DISCUSSION THREAD

**Discussion by felipemarins** [ppy/osu#26068] [Q&A] [answered]
I remember it being fixed in some release, but it seems it has returned somehow. It's nothing too serious, just a bit annoying that there is a 1px line appearing and disappearing as I scroll around the top of the profile screen.

![image](https://github.com/ppy/osu/assets/53406701/b395c2dc-5364-46dc-a9b6-4ea2a38f53b9)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/26077
